services/continuous_querier/service.go

```diff
@@ -153,6 +153,7 @@ func (s *Service) backgroundLoop() {
 			return
 		case <-s.RunCh:
 			if s.MetaStore.IsLeader() {
+				s.Logger.Print("running continuous queries by request")
```

Comment:
Since CQs are usually run by the service, I thought it would be good to log a message if they were triggered by an external request instead.